yellowbrick/features/decomposition.py

```diff
@@ -0,0 +1,66 @@
+##########################################################################
+## Imports
+##########################################################################
+
+from .base import FeatureVisualizer
+
+from sklearn.pipeline import Pipeline
+from sklearn.decomposition import PCA
+from sklearn.preprocessing import StandardScaler
+
+##########################################################################
+## Decomposition Feature Visualizer
+##########################################################################
+
+class ExplainedVariance(FeatureVisualizer):
+    """
+
+    Parameters
+    ----------
+    
+
+    Examples
+    --------
+
+    >>> visualizer = ExplainedVariance()
+    >>> visualizer.fit(X)
+    >>> visualizer.transform(X)
+    >>> visualizer.poof()
+
+    Notes
+    -----
+    
+    """
+
+    def __init__(self, ax=None, n_components=None, scale=True, center=True, colormap='RdBu_r', 
+                 **kwargs):
+
+        super(ExplainedVariance, self).__init__(ax=ax, **kwargs)
+
+        self.colormap = colormap
+        self.explained_variance_ = None
+        self.n_components = n_components
+        self.center = center
+        self.scale = scale
+        self.pipeline = Pipeline([('scale', StandardScaler(with_mean=self.center,
+                                                                   with_std=self.scale)), 
+                                                                  ('pca', PCA(n_components=self.n_components))])
+
+    def fit(self, X, y=None):
+        self.pipeline.fit(X)
+        return self
+
+    def transform(self, X):
+        self.draw(self.pipeline.steps[-1][1].explained_variance_)
```

Comment:
You have a property, `explained_variance_` but are not setting it; I recommend that you do and allow draw to access it directly rather than passing it into draw. Also, you're not doing anything on transform, so perhaps it would be better to put this into `fit()`? 

```python
def fit(self, X, y=None):
    self.pipeline.fit(X) 
    self.explained_variance_ = self.pipeline.steps[-1][1].explained_variance_ 
    self.draw() 
```

Another option is to simply make `explained_variance_` a property so that you always access it off of the pipeline, even during updates:

```python
@property 
def explained_variance_(self):
    return self.pipeline.steps[-1][1].explained_variance_ 
```

This has the advantage of ensuring that even if the visualizer is fit multiple times, the right visualization will be drawn.